github/event_types.go

```diff
@@ -130,19 +130,6 @@ type GollumEvent struct {
 	Installation *Installation `json:"installation,omitempty"`
 }
 
-// IssueActivityEvent represents the payload delivered by Issue webhook.
```

Comment:
> However, the tests of 3rd PR will fail until 1st is merged.

Actually, I just came up with a really sneaky way of getting around that. :stuck_out_tongue_winking_eye: